### DISCUSSION TITLE
What color paletter our website should use?

### DISCUSSION THREAD

**Discussion by JeanMontesSantiago** [uprm-inso4101-2023-2024-s2/semester-project-personalfinancemanager#160] [General]
Selecting a color palette for our website is a strategic decision, directly affecting the user interface (UI) and overall user experience (UX). This choice should reflect the core attributes of the website's entity and be designed to enhance usability and accessibility.

**Comment by diegoq48** [comment]
I'd reckon some colors that are easy on the eyes as to not cause stress but also allow certain aspects of the website to be highlighted to cause stress in a way like say have white and blue overall since these colors relax the eye but have a stronger color for say expenses.
